Requirements/ReqIFDal/ReqIfImportMappingManager.cs

```diff
@@ -252,19 +282,55 @@ private void NavigateToRequirementSpecificationMappingDialog()
 
             var thingFactory = new ThingFactory(this.iteration, this.datatypeDefinitionMap, typeMap, this.currentDomain, this.reqIf.Lang);
             thingFactory.ComputeRequirementThings(this.reqIf);
-
+            
             var dialog = new RequirementSpecificationMappingDialogViewModel(thingFactory, this.iteration, this.session, this.thingDialogNavigationService, this.reqIf.Lang);
             var res = (MappingDialogNavigationResult)this.dialogNavigationService.NavigateModal(dialog);
 
-            if (res == null || !res.Result.HasValue || !res.Result.Value)
+            if (res?.Result != true)
```

Comment:
Please use a more discriptive name for `res`, like mappingDialogNavigationResult